### ISSUE TITLE
图片显示问题

### ISSUE THREAD

**Issue by youngjuning** [theme-next/hexo-theme-next#394]
![image](https://user-images.githubusercontent.com/13204332/44500635-2505ce00-a6bc-11e8-83d4-eeeec98b2a6b.png)

**Comment by youngjuning** [comment]
这是一个可点击的图片，我不想让他是 fancybox，请问我该怎么写

**Comment by maple3142** [comment]
Do you mean how to prevent the img in an anchor from being wrapped as fancybox?

**Comment by youngjuning** [comment]
@maple3142 yes

**Comment by youngjuning** [comment]
![image](https://user-images.githubusercontent.com/13204332/44502142-a82a2280-a6c2-11e8-9e69-86032d5064dc.png)
当一篇文章同时属于两个分类，其中一个分类会是另一个分类的子分类。
![image](https://user-images.githubusercontent.com/13204332/44502168-cdb72c00-a6c2-11e8-9cd2-4feecdb66fc9.png)

**Comment by KaitoHH** [comment]
[**@youngjuning**](https://github.com/youngjuning) commented on [2018年8月23日 GMT+8 上午10:53](https://github.com/theme-next/hexo-theme-next/issues/394#issuecomment-415266117 "2018-08-23T02:53:53Z - Replied by Github Reply Comments"):
> [![image](https://user-images.githubusercontent.com/13204332/44502142-a82a2280-a6c2-11e8-9e69-86032d5064dc.png)](https://user-images.githubusercontent.com/13204332/44502142-a82a2280-a6c2-11e8-9e69-86032d5064dc.png)  
> 
> 当一篇文章同时属于两个分类，其中一个分类会是另一个分类的子分类。  
> 
> [![image](https://user-images.githubusercontent.com/13204332/44502168-cdb72c00-a6c2-11e8-9cd2-4feecdb66fc9.png)](https://user-images.githubusercontent.com/13204332/44502168-cdb72c00-a6c2-11e8-9cd2-4feecdb66fc9.png)

That is default behavior of Hexo. You can wrap these with brackets to define on the same hierarchical level.
```yml
categories:
  - [category1]
  - [category2]
```
refer [this](https://hexo.io/docs/front-matter.html) for detailed.

**Comment by Ahaochan** [comment]
+1, [My Blog](https://ahaochan.github.io/about/) has same issue.
Even if I use `<img>` Tag to replace `![](https://img.shields.io/github/stars/Ahaochan/Tampermonkey.svg)`, The `<img>` Tag still wrapped as fancybox.
```
<img src="https://img.shields.io/github/stars/Ahaochan/Tampermonkey.svg" alt="star" style="display: inline;margin-bottom: -1%;">
```

**Comment by Raincal** [comment]
@youngjuning @Ahaochan Fixed in #468
